tests/ui/associated-type-bounds/no-gat-position.stderr

```diff
@@ -3,6 +3,11 @@ error[E0229]: associated item constraints are not allowed here
    |
 LL |     fn next<'a>(&'a mut self) -> Option<Self::Item<'a, As1: Copy>>;
    |                                                        ^^^^^^^^^ associated item constraint not allowed here
+   |
+help: consider removing this associated item binding
```

Comment:
```suggestion
help: consider removing this associated item constraint
```

due to the removal of the line `&& let hir::AssocItemConstraintKind::Equality { term } = constraint.kind` in `prohibit_assoc_item_constraint`, the removal suggestion now also fires for constraints that aren't bindings like here.

Reply:
Thanks!